### DISCUSSION TITLE
Colour logo in header, turns blue

### DISCUSSION THREAD

**Discussion by HarryJulsing** [SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter#102] [Q&A] [answered]
Hi,

We want to show a png/jpg logo image in the header.

When using the following script in the left header template, the logo turns blue:

<a href='https://www.logo.nl/'>
    <img src='/polarion/icons/group/logo.png' 
         alt='logo' 
         style='height: 20px'>
</a>

Do you have any suggestions to fix this?

Thanks,
Harry

**Comment by grigoriev** [comment]
Could you please provide us with more details and your code?

**Comment by HarryJulsing** [comment]
Hi Sergey,

Thanks for your quick reply,

I have attached the total configuration and the logo.

The logo is stored at global level:
.polarion/icons

For the export we use the ProRail CSS and the Test Header and Footer.

Thanks,
Harry


________________________________

**Comment by grigoriev** [comment]
Hi @HarryJulsing,
I can not find any attachments.

**Comment by HarryJulsing** [comment]
[
![ProRailLogo](https://github.com/user-attachments/assets/8dc7e9ff-6916-404f-880d-0ae9b77cd61e)
[pdf-exporter-r5313.zip](https://github.com/user-attachments/files/16348643/pdf-exporter-r5313.zip)
](url)

**Comment by HarryJulsing** [comment]
Reply with attachments do not seem to work

**Comment by HarryJulsing** [comment]
Hi Sergey,
I added the attachments last week, were you able to do something with them?

Thanks,
Harry
________________________________

**Comment by grigoriev** [comment]
Hi @HarryJulsing,

the problem is related to that fact that WeasyPrint can not load image and in this case it renders it as alt text:
```html
<a href='https://www.prorail.nl/'>
    <img src='/polarion/icons/group/ProRailLogo.png' 
         alt='ProRail' 
         style='height: 50px'>
</a>
```
<img width="213" alt="image" src="https://github.com/user-attachments/assets/fbc96ff3-6aa3-4c4d-a712-092ef73df598">

If you are using this code I suppose everything works as expected:
```html
<img width="100" src="https://lumiguide.eu/wp-content/uploads/2024/03/prorail.png">
```
<img width="233" alt="image" src="https://github.com/user-attachments/assets/446e3195-8568-4011-8eb6-b5db46dae35a">


I've reproduce your issue only using [WeasyPrint as Service](https://github.com/SchweizerischeBundesbahnen/weasyprint-service) when the Polarion host can not be resolved in Docker container.

If WeasyPrint is not able to load the icon you can find the following log entries:
```log
2024-07-30 15:22:01 ERROR:weasyprint:Failed to load image at 'http://polarion-unresolvable-host/polarion/icons/group/ProRailLogo.png': URLError: <urlopen error [Errno -2] Name or service not known>
2024-07-30 15:22:02 ::ffff:192.168.65.1 - - [2024-07-30 13:22:02] "POST /convert/html?presentational_hints=true&pdf_variant=pdf%2Fa-2b HTTP/1.1" 200 119546 0.826479
```

If WeasyPrint Service is able to load the icon everything looks pretty good:
<img width="255" alt="image" src="https://github.com/user-attachments/assets/e2d3770b-8c55-43fc-b7db-31c5785d0e26">

Please let me know if it helps.

**Comment by grigoriev** [comment]
I have created the issue https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/issues/112

**Reply by grigoriev** [reply, marked as answer]
Hi @HarryJulsing,

could you please try with the just released PDF-Exporter v5.6.0 ?
It should solve the problem with image rendering in header/footer.

**Comment by HarryJulsing** [comment]
Hi Sergey,
This version has not been made available through the Polarion Extension site. The latest is V 5.1.2 .
How can I get that version, so we can install it on our own test environment first?
Thanks,
Harry
________________________________

**Reply by grigoriev** [reply]
Hi @HarryJulsing,

the latest version of the PDF-Exporter can be downloaded here: https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/packages/2175431
